README.md

```diff
@@ -182,6 +196,43 @@ This is useful if you have control over secret creation and would like to avoid
 
 Example: `buildkite-secret`
 
+### `external-secrets` (optional, string or array)
+
+Mount one or many secrets as a directory inside the container from an external source using [ExternalSecrets](https://external-secrets.io/v0.7.2/). Must be in the form `secretKey:externalProperty:externalKey`.
```

Comment:
Nit: Should likely not link to a specific version. I'd also mention that this is an third-party controller that has to be separately installed on the cluster.

Reply:
Fixed